### PR TITLE
Adds a database size report script and dedups common code

### PR DESCRIPTION
**Pull request by conbrad**
Makes it easier to check the size of database tables

### CODE REVIEW

File: api/scripts/database_size_report.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname "$0")"
+
+source "$SCRIPT_DIR/common_oc_checks.sh"
+
+# From https://wiki.postgresql.org/wiki/Disk_Usage
+SIZE_REPORT_QUERY="SELECT *, pg_size_pretty(total_bytes) AS total
+    , pg_size_pretty(index_bytes) AS index
+    , pg_size_pretty(toast_bytes) AS toast
```
Comment (andrea-williams) [inline api/scripts/database_size_report.sh:23]: what are toast bytes?

**Review by andrea-williams — APPROVED** [review]

File: api/scripts/database_size_report.sh (hunk 1 above)
Comment (conbrad) [reply api/scripts/database_size_report.sh:23]: Postgres does not support physical row data that crosses the size of a data block. To enable values larger than data blocks it breaks them up into smaller pieces and are stored in the TOAST table that is related to the original table.

https://wiki.postgresql.org/wiki/TOAST

Comment (Sybrand) [reply api/scripts/database_size_report.sh:23]: @conbrad could you add that comment into the script please?

**Review by Sybrand — APPROVED** [review]
Useful. If you could just add the toast comment into the code that would be nice.

**Pull request merged**